operations/service.go

```diff
@@ -108,7 +110,33 @@ func Service() cli.Command {
 			//
 			// starting grpc
 			//
-			rpcSrv := grpc.NewServer()
+			middlewareConf := gimlet.UserMiddlewareConfiguration{
+				// TODO: figure out correct input for these
+				CookieName:     "cookieToken",
+				HeaderUserName: "username",
+				HeaderKeyName:  "apiKey",
+			}
+			cedarConf, err := env.GetConf()
+			if err != nil {
+				return errors.Wrap(err, "problem getting env configuration")
+			}
+			var rpcSrv *grpc.Server
+			if cedarConf.UserManager != nil {
+				unary := aviation.MakeAuthenticationRequiredUnaryInterceptor(
+					cedarConf.UserManager,
+					middlewareConf,
+				)
+				stream := aviation.MakeAuthenticationRequiredStreamingInterceptor(
+					cedarConf.UserManager,
+					middlewareConf,
+				)
+				rpcSrv = grpc.NewServer(
+					grpc.UnaryInterceptor(unary),
+					grpc.StreamInterceptor(stream),
+				)
```

Comment:
I'd just merge all these statements and do it inline,

Reply:
done